UnitTests/UnitTests.csproj

```diff
@@ -192,6 +192,10 @@
     <Content Include="Assets\Wide310x150Logo.scale-200.png" />
   </ItemGroup>
   <ItemGroup>
```

Comment:
We should add a code block test since its new.  Anything else new?

Reply:
The original Code Block test still works.